engine/launcher/native-image-config/reflect-config.json

```diff
@@ -0,0 +1,88 @@
+[
```

Comment:
It may be in some specific cases - depends on the reflection usage (whether the program accesses reflectively stuff only on some platform).

In our case it seems it is not platform specific - the classes don't seem to be and the program builds and runs correctly on all platforms (I think, this will be verified once the CI succeeds; for sure before merging the PR it will be verified that it works).

The process that creates the file is described in the `native-image.md` documentation.

Reply:
Yeah. It just doesn't specify if the result file is platform specific.